quisp/modules/QRSA/RuleEngine/RuleEngine.cc

```diff
@@ -698,51 +671,104 @@ InterfaceInfo RuleEngine::getInterface_toNeighbor_Internal(int local_qnic_addres
 }
 
 void RuleEngine::scheduleFirstPhotonEmission(BSMtimingNotifier *pk, QNIC_type qnic_type) {
-  SchedulePhotonTransmissionsOnebyOne *st = new SchedulePhotonTransmissionsOnebyOne("SchedulePhotonTransmissionsOneByOne(First)");
-  if (ntable.empty())  // Just do this once, unless the network changes during the simulation.
+  if (ntable.empty()) {
     ntable = hardware_monitor->passNeighborTable();  // Get neighbor table from Hardware Manager: neighbor address--> InterfaceInfo.
+  }  // Just do this once, unless the network changes during the simulation.
 
+  photon_transmission_config ptc;
```

Comment:
`ptc` is a confusing name. how about `config` or `transimission_config`?

Reply:
Let me use `transmission_config` instead. Thanks